adapters/rubicon/rubicon.go

```diff
@@ -578,6 +590,7 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adap
 
 	requestImpCopy := request.Imp
 
+	rubiconRequest := request
```

Comment:
This will create a second pointer to the original request. I don't think you want that. Perhaps `rubiconRequest := *request` ?

Reply:
Done